locales/en/plugin__odf-console.json

```
@@ -666,8 +666,8 @@
   "Deploys MultiCloud Object Gateway without block and file services.": "Deploys MultiCloud Object Gateway without block and file services.",
   "Deploys Data Foundation with block, shared fileSystem and object services.": "Deploys Data Foundation with block, shared fileSystem and object services.",
   "Deployment type": "Deployment type",
-  "Set Ceph RBD as the default StorageClass": "Set Ceph RBD as the default StorageClass",
-  "Configure a default RBD StorageClass to eliminate manual annotations within a StorageClass or selecting a specific StorageClass when making storage requests or provisions in your PVCs.": "Configure a default RBD StorageClass to eliminate manual annotations within a StorageClass or selecting a specific StorageClass when making storage requests or provisions in your PVCs.",
+  "Use Ceph RBD as the default StorageClass": "Use Ceph RBD as the default StorageClass",
+  "Configure default RBD StorageClass to avoid adding manual annotations within a StorageClass and selecting a specific StorageClass when making storage requests or provisions in your PVCs.": "Configure default RBD StorageClass to avoid adding manual annotations within a StorageClass and selecting a specific StorageClass when making storage requests or provisions in your PVCs.",
```

```suggestion
  "Configure default RBD StorageClass to avoid adding manual annotations within a StorageClass and selecting a specific StorageClass when making storage requests or provisions in your PVCs.": "Configure default RBD StorageClass to avoid adding manual annotations within a StorageClass or selecting a specific StorageClass when making storage requests or provisions in your PVCs.",
```
@Chandan141996 it should be `or`.
Also, can we remove `selecting a specific StorageClass when making storage requests or provisions in your PVCs.` part entirely ??

---

The same or similar was mentioned in the epic, would advise not making any last-minute changes unless its important or has high consequences of user experience @SanjalKatiyar

---

it definitely won't have high impact !!
so if this is something which needs to be gone through doc team first and will probably take a while, then it is okay to ignore and do not block this PR... otherwise, good to make it `or`...

---

Yeah, I remember now, Disha changed it since these are two separate actions, that the user can perform in two different places in the UI. Hence replaced the 'or' with an 'and' in the original draft of the message.